### PR TITLE
add piggyback timing to parameters.js

### LINKED ISSUE #100: Force piggybacking.
I can't see a way to force piggybacked responses, which is desirable for working with motes (by reducing the amount of packets and thus the probability of loss).

Is it possible?

### PR DESCRIPTION
**Pull request by neophob**
should fix #100

### CODE REVIEW

**Comment by mcollina** [comment]
I would put this in the server options, rather than in the parameters. This should be really easy to change and customize for each server.

**Comment by neophob** [comment]
make sense, ill update the pr

**Comment by neophob** [comment]
done, the piggyback timeout can be defined as server option, example:
 ```coap.createServer({ piggybackReplyMs: 200 })```

File: test/server.js [hunk 1]
```diff
@@ -946,7 +946,20 @@ describe('server', function() {
...
+
+describe('piggyback timing option', function() {
+  it('send message', function(done) {
+    var piggyBackTimeout = 42
+
+    var server = coap.createServer({ piggybackReplyMs: piggyBackTimeout })
+
+    expect(server._options.piggybackReplyMs).to.eql(piggyBackTimeout)
+    server.close()
+    done()
```
Comment (mcollina) [inline test/server.js]: It would be better if we test this with a `setTimeout`, and add a test also for the default.

**Comment by neophob** [comment]
Yes, testing was done a bit in a hurry. fixed it now and improved parameter check

**Comment by mcollina** [comment]
Thanks, release as 0.13.1.

**Pull request merged**